src/nanovg_gl.h

```diff
@@ -363,38 +364,54 @@ static GLNVGtexture* glnvg__allocTexture(GLNVGcontext* gl)
 			gl->textures = textures;
 			gl->ctextures = ctextures;
 		}
-		tex = &gl->textures[gl->ntextures++];
+		idx = gl->ntextures++;
```

Comment:
Instead of using ``rand()`` for the salt below, if you're here you know you're initializing a new entry for which you can set salt = 0.